src/main/scala/logger/records/data/FunctionRecord.scala

```diff
@@ -12,9 +12,8 @@ import viper.silicon.state.State
 import viper.silicon.state.terms.Term
 import viper.silver.ast
 
-class FunctionRecord(v: ast.Function, s: State, p: PathConditionStack) extends MemberRecord {
+class FunctionRecord(v: ast.Function, p: PathConditionStack) extends MemberRecord {
```

Comment:
Why is the `State` (here and further down) no longer relevant?

Reply:
I have no opinion on whether it's useful, but currently it was set to `null` in all paths.